packages/lex-cli/src/codegen/server.ts

```diff
@@ -164,8 +164,10 @@ function genNamespaceCls(file: SourceFile, ns: NsidNS) {
     })
     method.setBodyText(
       [
-        `/** @ts-ignore */`,
-        `return this.server.xrpc.method('${schema.id}', handler)`,
+        // Placing schema on separate line, since the following one was being formatted
+        // into multiple lines and causing the ts-ignore to ignore the wrong line.
+        `const schema = '${schema.id}' // @ts-ignore`,
```

Comment:
This was an incidental change that I found I had to make to avoid type errors.  I tried for the alternate path of ensuring the generated file would not format the following line onto multiple lines when it got too long.  But I wasn't able to easily identify where that formatting was happening— I think it may be something implicit in ts-morph.